pkg/servicequota/vpcusage.go

```diff
@@ -0,0 +1,71 @@
+package servicequota
```

Comment:
We should create a function for creating a new quota checker, e.g.:

```go
func NewVpcChecker(provider v1alpha1.CloudProvider) *VpcChecker {
  return &VpcChecker{
    provider: provider,
  }
}
```

Or something to this effect.

Reply:
Done :)